embulk-core/src/main/java/org/embulk/deps/DependencyCategory.java

```diff
@@ -0,0 +1,23 @@
+package org.embulk.deps;
+
+public enum DependencyCategory {
+    CLI("CLI", "Embulk-Resource-Class-Path-Cli"),
+    MAVEN("Maven", "Embulk-Resource-Class-Path-Maven"),
```

Comment:
It's worth having a simple test case just as a design contract, in case these are changed in gradle file.

Reply:
Sounds reasonable. Added in 7734acb4fb05f0ae14c1beaa9406ed838e13b059